mtg_card_identifier/CardScanner/CardScannerFile.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  CardScannerFile.swift
+//  mtg_card_identifier
+//
+//  Created by mac on 8/16/23.
+//
+
+import SwiftUI
+import UIKit
+import VisionKit
+
+struct LiveTextScanner: UIViewControllerRepresentable {
+    
+    // Will need more than just a string when we are doing the api call.
+    var scanedText: Binding<[String]>
+    var style: Style
+    
```

Comment:
Let’s add a`var showTextBoundingRect = false` here and use it on the DataScannerViewController isHighlightingEnbabled initializer.